playlistmanager.lua

```diff
@@ -403,6 +405,9 @@ function on_end_file()
   path = nil
   directory = nil
   filename = nil
+  if settings.close_playlist_on_playfile then
+    remove_keybinds()
+  end
```

Comment:
I think this shouldn't be in this function right? This event is fired even when user doesn't actively change the file.

The setting `playfile` is referring to the action of selecting a file to play, rather than a file just being played or loaded.